joern-cli/frontends/gosrc2cpg/src/test/scala/io/joern/go2cpg/passes/ast/ASTCreationForStructureMemberTests.scala

```diff
@@ -10,7 +10,7 @@ import io.shiftleft.semanticcpg.language.operatorextension.OpNodes
 import scala.collection.immutable.List
 import io.joern.gosrc2cpg.astcreation.Defines
 
-class ASTCreationForStructureTests extends GoCodeToCpgSuite {
+class ASTCreationForStructureMemberTests extends GoCodeToCpgSuite {
```

Comment:
Rename this file to `TypeDeclMembersTest` we have removed the prefix for other files

Reply:
Addressed here: https://github.com/joernio/joern/pull/3585/commits/6461283b1309525a09028e9c7812f0602c880e34